internal.nix

```diff
@@ -416,6 +429,7 @@ rec {
       shellHook = ''
         # FIXME: we should somehow register a GC root here in case of a symlink?
         ${add_node_modules_to_cwd nm node_modules_mode}
+        npm run install # run install script for the root package
```

Comment:
fixme `missing script: install`

we need to run the scripts postinstall, prepare, ... - if they exist